ComSemApp/student/views.py

```diff
@@ -891,10 +884,10 @@ def score_generator(self, transcription : str, expression : Expression) -> Itera
         yield (score, None)
         for attempt in correct_attempts:
             if score == MAX_SCORE:
-                raise StopIteration
+                return
```

Comment:
Manually raising the StopIteration was causing some issues, so I'm fixing that here